lib/rubocop/cop/lint/no_vcr_recording.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+require 'unparser'
+
+module RuboCop
+  module Cop
+    module Lint
+      # @example
+      #   # bad
+      #   vcr: { record_mode: :once, tag: :foo }
+      #   vcr: { record_mode: :none }
+      #
+      #   # good
+      #   vcr: { tag: :foo }
+      #   vcr: true
+      class NoVCRRecording < RuboCop::Cop::Base
+        extend AutoCorrector
+
+        MSG = 'Do not set :record option in VCR'.freeze
```

Comment:
My only question would be should we provide a "why"? But then other cops don't do this, so maybe not?

```suggestion
        MSG = "Do not set :record option in VCR because we shouldn't record new cassettes in CI/CD".freeze
```